.env.example

```diff
@@ -63,3 +63,6 @@ CLIENT_SECRET_GITHUB_LOGIN=
 
 CLIENT_ID_GITLAB_LOGIN=
 CLIENT_SECRET_GITLAB_LOGIN=
+
+OTEL_COLLECTOR_OTLP_URL=
+OTEL_TELEMETRY_COLLECTION_ENABLED=
```

Comment:
Is there a reason we explicitly want this enabled/disabled env? What if we determined if it is enabled by checking if  OTEL_COLLECTOR_OTLP_URL is set?

Reply:
so currently I added support for both OTLP export and prometheus-native export. If OTEL_TELEMETRY_COLLECTION_ENABLED is set to true but OTEL_COLLECTOR_OTLP_URL is not defined, it will default to the prometheus export else it uses OTLP

Reply:
it's better that we have support for multiple export methods so that we can test out which one is the most ideal in our setup (and for those who want to self-host, they can choose what's ideal for them as well)

Reply:
okay, i understand. 

nit: I would rename `OTEL_COLLECTOR_OTLP_URL` to `OTEL_EXPORTER_OTLP_ENDPOINT`

Instead defaulting to promethus, I think users should intentionally set the export type. What do you think about setting `OTEL_EXPORT_TYPE=otlp` `OTEL_EXPORT_TYPE=prometheus` and we do a switch over this? This way, if we want to add other export type, we can easily do that in the future too?

Reply:
yea that makes more sense, will adjust accordingly